File: corpus/2018q3/_p2q2_ok/_check/D;0533.java

import java.util.Scanner;
public class Questao2 {

    public static void main(String[] args) {
    int A [][];
    int i, j, n, v, r = 0, c = 0;
    int somap = 0, somab = 0;
    Scanner entrada = new Scanner (System.in);
    n = entrada.nextInt();
    A = new int [n][n];

    for (j = 0; j < n; j++){
        for (i = 0; i < n; i++){

            v = entrada.nextInt();
            A[i][j] = v;

            if (c == 0){
                somap = somap + v;
		System.out.println("preta");
                        }
            else{
                if (c%2 == 0){
                    somap = somap + v;
		    System.out.println("preta");
			}
                else{
                    somab = somab + v;
		    System.out.println("branca");
                }
            }
            c = c + 1;
            }    
        }
        if (somap > somab){
        r = 0;
        }
        if (somab > somap){
        r = 1;
        }
        if (somab == somap){
        r = 2;
        }

    System.out.println(somap + " " + somab + " " + r);
    } 
}

File: corpus/2018q3/_p2q2_ok/_check/D;0533.py
int A [][]
int i, j, n, v, r = 0, c = 0
int somap = 0, somab = 0
Scanner entrada = new Scanner (System.in)
n = int(input())
for 0 in range(0, n):
    for 0 in range(0, n):
        v = int(input())
        if c == 0:
            somap = somap + v
            print("preta")
        else:
            if c%2 == 0:
                somap = somap + v
                print("preta")
            else:
                somab = somab + v
                print("branca")
        c = c + 1
if somap > somab:
    r = 0
if somab > somap:
    r = 1
if somab == somap:
    r = 2
print(somap + " " + somab + " " + r)